splice_machine/src/main/java/com/splicemachine/derby/procedures/ShowCreateTableProcedure.java

```diff
@@ -269,14 +267,20 @@ private static String createConstraint(TableDescriptor td, String schemaName, St
                         break;
                     keyColumns = cd.getKeyColumns();
                     boolean uniqueFirstCol = true;
+                    StringBuffer uniqueStr = new StringBuffer();
+                    StringBuffer columnNames = new StringBuffer();
+                    String constraintName = cd.getConstraintName();
                     for (int index=0; index<keyColumns.length; index ++) {
                         String colName = columnDescriptorMap.get(keyColumns[index]).getColumnName();
-                        uniqueStr.append(uniqueFirstCol ? ", CONSTRAINT " + cd.getConstraintName() +
-                                " UNIQUE (\"" + colName + "\"": ",\"" + colName + "\"");
+                        uniqueStr.append(uniqueFirstCol ? ", CONSTRAINT " + constraintName + " UNIQUE (\"" +
+                                colName + "\"": ",\"" + colName + "\"");
+                        columnNames.append(uniqueFirstCol ? "\"" + colName + "\"": ",\"" + colName + "\"");
```

Comment:
It looks like `uniqueFirstCol ? "\"" + colName + "\"": ",\"" + colName + "\""` can be extracted to a local variable to remove code duplication. But it's a minor thing.